Scripts/BuildPhases/LintNestedFrameworks.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -eu
+
+set pipefail
```

Comment:
I only now notice that the shebang doesn't use Bash. Possibly why I ShellCheck warned me about having `pipefail` in there.